en/deploy-br-federation-manager.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Deploy BR Federation Manager on Kubernetes
+summary: Learn how to deploy BR Federation Manager on Kubernetes.
+aliases: ['/docs/tidb-in-kubernetes/dev/deploy-br-federation-manager/']
+---
+
+# Deploy BR Federation Manager on Kubernetes
+
+This document describes how to deploy BR Federation Manager across multiple Kubernetes.
+
+## Prerequisites
+
+Before deploy BR Federation Manager on Kubernetes, you should satisfy conditions below:
+
+* Kubernetes >= v1.12
+* you have multiple Kubernetes clusters
+* All the Kubernetes clusters that serve as data planes have deployed TiDB-Operator
+
+## Deploy BR Federation Manager
```

Comment:
```suggestion
```

This heading feels somehow redundant and adds depth to the document structure.

You can start directly from `## Step 1: Generate Kubeconfig file in all Data Planes`.